### PR TITLE
add ttl index annotation support

### PR DESCRIPTION
**Pull request by krasaev**

### CODE REVIEW

**Comment by sulaukad** [comment]
@rashtao Please, take a look:)

**Comment by rashtao** [comment]
Hi @krasaev 

Many thanks for contributing. We are obliged by law to have a signed scanned copy of a Contributor License Agreement (CLA) to be able to accept pull requests to the official ArangoDB repository - even for such small changes.

We use an Apache 2 CLA for ArangoDB and its companion projects, which can be found here: https://www.arangodb.com/documents/cla.pdf Just fill in the form, sign and send a scanned copy over to cla@arangodb.com

Thank you

**Review by rashtao — APPROVED** [review]

**Comment by joerg84** [comment]
@sulaukad Did you see the note about the CLA. This is great work and we would love to merge it :-)

**Comment by krasaev** [comment]
> @krasaev Did you see the note about the CLA. This is great work and we would love to merge it :-)

most work was done by @rashtao =)

I've sent the agreement. Thank you.

File: src/main/java/com/arangodb/springframework/core/mapping/DefaultArangoPersistentEntity.java [hunk 1]
```diff
@@ -20,23 +20,28 @@
 
 package com.arangodb.springframework.core.mapping;
 
-import java.lang.annotation.Annotation;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.HashMap;
-import java.util.List;
```
Comment (krasaev) [inline src/main/java/com/arangodb/springframework/core/mapping/DefaultArangoPersistentEntity.java:28]: nit. would be nice to have style guide

**Comment by joerg84** [comment]
Received the CLA, excited about your contribution.
@rashtao FYI

File: src/main/java/com/arangodb/springframework/core/mapping/DefaultArangoPersistentEntity.java (hunk 1 above)
Comment (rashtao) [reply src/main/java/com/arangodb/springframework/core/mapping/DefaultArangoPersistentEntity.java:28]: I agree, it's on my list!

**Pull request merged**